apps/mobile-wallet/src/hooks/layout/useAutoScrollOnDragEnd.tsx

```diff
@@ -16,17 +16,20 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why is this needed?

Reply:
if it's really needed, maybe it can just be moved to the line where it is needed instead of marking the whole file?

Reply:
Interesting how this is not needed in `FlatList`:

```ts
export class FlatList<ItemT = any> extends FlatListComponent<
  ItemT,
  FlatListProps<ItemT>
> {}
```

Reply:
Moved to the line instead of marking the whole file (in the follow up PR)